app/com/arpnetworking/play/PillarModule.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2017 Smartsheet.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.play;
```

Comment:
My hope was to pull this out into a Cassandra/Pillar Play module.  I'll move it to global for now, but probably put it in play when/if I pull it out.